test/evergreen/compatibility_test_for_mongodb_releases.sh

```diff
@@ -5,6 +5,18 @@
 
 set -e
 
+#############################################################
+# format_b_flag:
+#       arg1: release
+#############################################################
+format_b_flag()
+{
+        # Return if the release's format command takes the -B flag for backward compatibility.
+        test "$1" = "develop" && return 0
+        test "$1" = "mongodb4.4" && return 0
```

Comment:
Not sure why the `-B` arg setting is disabled for `develop` and `mongodb-4.4` branches?
Btw, `mongodb4.4` --> `mongodb-4.4`.

Reply:
It's actually enabled on `develop` and `mongodb-4.4`, they're they only branches that support the `-B` option to format.

Thanks for the typo, fixed in 74f019c85.